modelforge/curation/spice_114_curation.py

```diff
@@ -142,6 +147,26 @@ def _init_record_entries_series(self):
             "wiberg_lowdin_indices": "series_atom",
         }
 
+    def _calculate_reference_charge(self, smiles: str) -> unit.Quantity:
+        """
+        Calculate the total charge of a molecule from its SMILES string.
+
+        Parameters
+        ----------
+        smiles: str, required
+            SMILES string of the molecule.
+
+        Returns
+        -------
+        total_charge: unit.Quantity
+        """
+
+        from rdkit import Chem
+
+        rdmol = Chem.MolFromSmiles(smiles, sanitize=False)
+        total_charge = sum(atom.GetFormalCharge() for atom in rdmol.GetAtoms())
+        return total_charge * unit.elementary_charge
```

Comment:
can we cast this to int?

Reply:
That would probably be safer.  I've updated it in all the spice curation files